### ISSUE TITLE
Feature suggestion : web app capabilities

### ISSUE THREAD

**Issue by robindevouge** [maeligg/my-online-cookbook#15]
You know, the cool feature when you can add it to you homescreen with a nice icon instead of the letter in the square, and the navbar is hidden 😊

**Comment by maeligg** [comment]
Good idea, would be nice to have the recipes accessible offline, too. I'll look into it.
